integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -259,3 +263,49 @@ def get_drafted_pages(
             "drafted_pages": drafted_pages,
             "single_drafted_page": single_drafted_page,
         }
+
+    @json_response
+    # pylint: disable=unused-argument, disable=no-self-argument
+    def get_translation_coverage_context(
+        request: HttpRequest, region_slug: str
+    ) -> JsonResponse:
+        r"""
+        Extend context by info on translation coverage of pages
+
+        :return: Dictionary containing the context for translation coverage of pages in a region
+        """
+        number_of_outdated_pages = 0
+
+        if not request.region.default_language:
+            return {}
+
+        languages = LanguageTreeNode.objects.filter(region=request.region).exclude(
+            language=request.region.default_language
+        )
+
+        pages_in_region = Page.objects.filter(region=request.region)
```

Comment:
At the moment explicit and implicit archived pages are also counted into the pages. I think it's more consistent with the Translation Report to exclude them.

Reply:
Otherwise the calculation looks fine

Reply:
Oh, very good point! Thanks :blush: